randovania/games/fusion/logic_database/Sector 3 PYR.txt

```diff
@@ -548,13 +548,17 @@ Extra - room_id: [14]
   * L0 Hatch to Sova Suite/Door to Pyrochamber Access
   * Extra - door_idx: (28,)
   > Door to Pyrochamber
-      All of the following:
-          # TODO: Possibly add singlewall jump trick difficulty with jump extend as an option if singlewall jump is kept.
-          Any of the following:
-              Space Jump
-              All of the following:
-                  Can Freeze Enemies
-                  High Jump or Wall Jump (Beginner)
+      Any of the following:
+          Space Jump
+          All of the following:
+              Can Freeze Enemies
+              High Jump or Wall Jump (Beginner)
+          All of the following:
+              # Jump Extend SWJ Method
+              Jump Extends (Advanced)
+              Any of the following:
+                  Wall Jump (Advanced)
+                  High Jump and Wall Jump (Intermediate)
```

Comment:
A) use single wall jump template
b) i'm unsure i follow on this one. dont you need some way to destroy the wall-guys? I dont think you can jump high enough otherwise to jump over them to the wall to WJ. You got some videos for this?

Reply:
With single wall jumps you would need to have jump extends and to continue the single wall jump on the opposite wall to get enough height. I can find a clip from a recent stream in which I did this while testing. 

Are we leaving single walljump tricks under the wall jump trick umbrella? The main reason I created these TODOs in the first place was due to there not being definitive direction one way or the other yet on whether single wall jumping is being kept in. If it is going to kept, it might make sense to have SWJ as it's own trick category instead of lumping it together with wall jumps.

The only thing that is unclear in this logic is the WJ or HJ and WJ and why they would be different difficulty levels.

Reply:
RE: Herc's last point, does HJ affect height coming out of a Wall Jump?
HJ/HJless difficulty would only need to be different if that was the case. (more jumps means more harder)

Reply:
Yeah, the only difference is the height off the wall jump. With HJ you can be a bit lower and without HJ it's one or two more jumps to get the height and jump extends to make this work.

Reply:
HJ only changes the initial jump height, it doesn't impact the height from wall jumps. That said, with HJ, it's 2 wall jumps to get the right height and without HJ it's 3 (plus wall jumping at the right place to avoid bonking the ceiling)

There is another way to do this jump without the jump extend, which is turning SA off in mid-air before the last wall jump.